### PR TITLE
Remove important data from sentry

### PR DESCRIPTION
**Pull request by FabianDevel**

### CODE REVIEW

File: app/src/main/java/com/infomaniak/drive/data/api/UploadTask.kt [hunk 1]
```diff
@@ -162,7 +159,7 @@ class UploadTask(
                         continue
                     }
 
-                    SentryLog.i("kDrive", "Upload > ${uploadFile.fileName} chunk:$chunkNumber has permission")
+                    SentryLog.i("kDrive", "Upload > File chunks number :$chunkNumber has permission")
```
Comment (KevinBoulongne) [inline app/src/main/java/com/infomaniak/drive/data/api/UploadTask.kt:162]:
```suggestion
                    SentryLog.i("kDrive", "Upload > File chunks number: $chunkNumber has permission")
```

File: app/src/main/java/com/infomaniak/drive/data/api/UploadTask.kt [hunk 2]
```diff
@@ -173,7 +170,7 @@ class UploadTask(
                     data = if (count == chunkSize) data else data.copyOf(count)
 
                     val url = uploadFile.uploadUrl(chunkNumber = chunkNumber, currentChunkSize = count)
-                    SentryLog.d("kDrive", "Upload > Start upload ${uploadFile.fileName} to $url data size:${data.size}")
+                    SentryLog.d("kDrive", "Upload > Start upload file to $url data size:${data.size}")
```
Comment (KevinBoulongne) [inline app/src/main/java/com/infomaniak/drive/data/api/UploadTask.kt:173]:
```suggestion
                    SentryLog.d("kDrive", "Upload > Start upload file to $url (data size:${data.size})")
```

File: app/src/main/java/com/infomaniak/drive/data/api/UploadTask.kt [hunk 3]
```diff
@@ -284,26 +277,17 @@ class UploadTask(
...
             uploadFile.resetUploadTokenAndCancelSession()
-            Sentry.withScope { scope ->
-                scope.setExtra("data", gson.toJson(uploadFile))
-                scope.setExtra("file size", "${uploadFile.fileSize}")
-                scope.setExtra("uploaded size", "$previousChunkBytesWritten")
-                scope.setExtra("bytesWritten", "$bytesWritten")
-                scope.setExtra("contentLength", "$contentLength")
-                scope.setExtra("chunk size", "$chunkSize")
-                Sentry.captureMessage("Chunk total size exceed fileSize 😢")
-            }
```
Comment (KevinBoulongne) [inline app/src/main/java/com/infomaniak/drive/data/api/UploadTask.kt:303]: Why remove this whole Sentry ?

File: app/src/main/java/com/infomaniak/drive/data/cache/FileController.kt [hunk 4]
```diff
@@ -810,7 +810,7 @@ object FileController {
                     updateFile(folder.id, realm) { file -> file.responseAt = apiResponse.responseAt }
                 } else {
                     Sentry.withScope { scope ->
-                        scope.setExtra("data", apiResponse.toString())
+                        scope.setExtra("Api responseAt", apiResponse.responseAt.toString())
```
Comment (KevinBoulongne) [inline app/src/main/java/com/infomaniak/drive/data/cache/FileController.kt:813]:
```suggestion
                        scope.setExtra("API responseAt", apiResponse.responseAt.toString())
```

File: app/src/main/java/com/infomaniak/drive/data/cache/FileController.kt [hunk 5]
```diff
@@ -822,7 +822,7 @@ object FileController {
                 updateFile(folder.id, realm) { file -> file.responseAt = apiResponse.responseAt }
             } else {
                 Sentry.withScope { scope ->
-                    scope.setExtra("data", apiResponse.toString())
+                    scope.setExtra("Api responseAt", apiResponse.responseAt.toString())
```
Comment (KevinBoulongne) [inline app/src/main/java/com/infomaniak/drive/data/cache/FileController.kt:825]:
```suggestion
                    scope.setExtra("API responseAt", apiResponse.responseAt.toString())
```

**Review by KevinBoulongne — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/infomaniak/drive/data/api/UploadTask.kt (hunk 3 above)
Comment (FabianDevel) [reply app/src/main/java/com/infomaniak/drive/data/api/UploadTask.kt:303]: Because it never triggered

**Review by KevinBoulongne — CHANGES_REQUESTED** [review]

**Review by KevinBoulongne — APPROVED** [review]

**Pull request merged**